node/core/approval-voting/src/lib.rs

```diff
@@ -119,6 +116,8 @@ const LOG_TARGET: &str = "parachain::approval-voting";
 pub struct Config {
 	/// The column family in the DB where approval-voting data is stored.
 	pub col_data: u32,
```

Comment:
One possible thing is to have a more consistent naming like `col_approval_data` and `col_session_data`.